src/loaders/TiffLoader.ts

```diff
@@ -135,21 +139,29 @@ class TiffLoader extends ThreadableVolumeLoader {
 
   private async loadOmeDims(): Promise<OMEDims> {
     if (!this.dims) {
-      const tiff = await fromUrl(this.url, { allowFullFile: true }).catch(
+      const tiff = await fromUrl(this.url, { allowFullFile: true }).catch<GeoTIFF>(
         wrapVolumeLoadError(`Could not open TIFF file at ${this.url}`, VolumeLoadErrorType.NOT_FOUND)
       );
       // DO NOT DO THIS, ITS SLOW
       // const imagecount = await tiff.getImageCount();
       // read the FIRST image
       const image = await tiff
         .getImage()
-        .catch(wrapVolumeLoadError("Failed to open TIFF image", VolumeLoadErrorType.NOT_FOUND));
-
-      const tiffimgdesc = prepareXML(image.getFileDirectory().ImageDescription);
-      const omeEl = getOME(tiffimgdesc);
-
-      const image0El = omeEl.getElementsByTagName("Image")[0];
-      this.dims = getOMEDims(image0El);
+        .catch<GeoTIFFImage>(wrapVolumeLoadError("Failed to open TIFF image", VolumeLoadErrorType.NOT_FOUND));
+
+      const omeEl = getOME(image.getFileDirectory().ImageDescription);
+
+      if (omeEl !== undefined) {
+        const image0El = omeEl.getElementsByTagName("Image")[0];
+        this.dims = getOMEDims(image0El);
+      } else {
+        console.warn("Could not read OME-TIFF metadata from file. Doing our best with base TIFF metadata.");
+        this.dims = new OMEDims();
+        this.dims.sizex = image.getWidth();
+        this.dims.sizey = image.getHeight();
+        this.dims.pixeltype = getPixelType(image.getBytesPerPixel());
+        this.dims.channelnames = ["Channel"];
```

Comment:
was there any metadata to be gleaned from the PerkinElmer xml in particular?  Info we could stuff in the metadata panel?  This would probably make the CellPainting folks happy.  We will have to ask them about that when they're here...